xml/pop_banners.xml

```diff
@@ -217,7 +217,58 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
   <!--
-
+<fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>matches to CMailServer</description>
```

Comment:
You can remove "matches to". In my opinion simple strings with the product name are best.  There are times when there are multiple fingerprints where adding simple comment helps describe the differences.  In this case I would simply change this to 'CMailServer POP3'